Mathlib/RingTheory/Kaehler/Localization.lean

```diff
@@ -0,0 +1,143 @@
+/-
+Copyright (c) 2025 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.RingTheory.Presentation
+
+/-!
+# Cotangent and localization away
+
+In this file we use the Jacobi-Zariski sequence to show some results
```

Comment:
Are you planning to add more results to this file? I don't think this file currently imports the Jacobi-Zariski file.